README.md

```diff
@@ -108,6 +108,35 @@ steps:
       release-branches: 'releases'
 ```
 
+## Create a pre-release release to a channel (e.g. "beta")
+
+For pre-releases it is recommended to create a separate workflow: (e.g.
+continuous-delivery-beta.yaml) which will run only on `beta` branch:
+
+```yaml
+on:
+  push:
+    branches:
+      - beta
+
+steps:
+  - name: Beta release
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    uses: ridedott/release-me-action@master
+    with:
+      release-branches: '[{"name":"beta","prerelease":true}]',
```

Comment:
What about setting the channel to beta? would that work or dont we use channels anymore?

Reply:
So channel is an abstraction around `npm` publishing, so that's why we can't use it in "Beta release", and it should be used as `Release` step like: `npm publish --tag=beta` where tag will create this "channel"